autobuild/templates/dockercompose.j2

```diff
@@ -11,11 +11,9 @@ services:
 {% if deploy_nevm and daemon.name == 'SYS' %}
       - 8545
       - 30303
-    ports:
-      - "8369:8369"
-      #- "8545:8545"
-      #- "30303:30303"
 {% endif %}
+    ports:
+      - "{{ daemon.p2pPort }}:{{ daemon.p2pPort }}"
```

Comment:
Hmm... this is going to create multiple `ports:` statements for SYS. I hope that doesn't break anything. 🤞

Reply:
Au contraire, Lord Percy. Mine is a plan so cunning you could pin a tail on it and call it a weasel; behold:
```
 SYS:
    image: blocknetdx/syscoin4:v4.3.0
    restart: unless-stopped
    expose:
      - 8370
      - 8369
      - 8545
      - 30303
    ports:
      - "8369:8369"
    entrypoint: /opt/blockchain/start-syscoin.sh
    command:
      - syscoind
...
```

Reply:
Wow! Magic!!!  I'm still trying to figure out why a second `ports:` isn't printed there! 🤔